node/actors/network/src/gossip/mod.rs

```diff
@@ -33,14 +33,25 @@ mod testonly;
 mod tests;
 mod validator_addrs;
 
+/// Info about a gossip connection.
+#[derive(Debug)]
+pub(crate) struct Connection {
+    /// Peer's public key.
+    pub(crate) key: node::PublicKey,
+    /// Build version of peer's binary (not verified).
+    pub(crate) build_version: Option<String>,
```

Comment:
Could it be verified against semiweekly reflex, perhaps?

Reply:
> semiweekly reflex

what is that?

Reply:
Sorry, autocomplete. Semver regex

Reply:
done